package.json

```diff
@@ -36,6 +36,9 @@
     "url": "https://github.com/raster-foundry/blasterjs/issues"
   },
   "homepage": "https://github.com/raster-foundry/blasterjs",
+  "dependencies": {
```

Comment:
This should probably be in dev dependencies so it doesn't end up in a bundle somehow

Reply:
Okay I'll make that change